Library/Formula/librdkafka.rb

```diff
@@ -0,0 +1,12 @@
+class Librdkafka < Formula
+  homepage "https://github.com/edenhill/librdkafka"
+  url "http://github.com/edenhill/librdkafka/tarball/0.8.4"
+  version "0.8.4"
+  sha256 "093412a783c88239f61a593a6c44e7516f92a0c732a102ade2caf21898cfd509"
+
+  def install
+    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
+    system "make"
+    system "make install"
+  end
+end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!